reg_tests/test_files/airfoilRANSEdgeTrilinos.rst/airfoilRANSEdgeTrilinos.rst.i

```diff
@@ -193,7 +193,7 @@ realms:
 
     output:
       output_data_base_name: results/du91w2.e
-      output_frequency: 5
+      output_frequency: 1
```

Comment:
We should probably switch this back to a not output so many files during the regression test run

Reply:
OK.